universal-app.md

```diff
@@ -2,9 +2,9 @@
 
 Follow these instructions to build your first GeoExt 3 application, using Sencha `cmd`. Supported versions:
 
-* Sencha Cmd v6.0.1.76 and v6.0.2.14
-* ExtJS GPL 6.0.0 (ext.version.number=6.0.0.640) and ExtJS GPL 6.0.1 (ext.version.number=6.0.1.250)
-* GeoExt 3 (from 2015-10-03 and from 2016-08-22)
+* Sencha Cmd v6.2.0.103
+* ExtJS GPL 6.2.0 (ext.version.number=6.2.0.???)
```

Comment:
sry. that was a placeholder before i left to do something less important... :dancing_women:

Real value should be `6.2.0.981`